### ISSUE TITLE
[Mobile Payments] Card Present Payment modals are not adapting well to large accessibility sizes

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#15321]
#### The Save receipt and continue button shown when payment is complete cropped at larger sizes.

![Image](https://github.com/user-attachments/assets/08f94d2b-238f-4efa-85e2-04ddd23deab9)

#### No scrolling on location services modal, doesn’t work well at larger dynamic type sizes and the button is invisible.

![Image](https://github.com/user-attachments/assets/d40108f7-2319-45fb-861d-f11172bbec73)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
